test-suite/squidconf/time_units

```diff
@@ -14,35 +14,35 @@
 dns_timeout 1 millisecond
 dns_timeout 0.001 second
 
-# maximum checks
-dns_timeout 9223372036854 milliseconds
-dns_timeout 9223372036 seconds
-dns_timeout 153722867 minutes
-dns_timeout 2562047 hours
-dns_timeout 106751 days
-dns_timeout 15250 weeks
-dns_timeout 7625 fortnights
-dns_timeout 3558 months
-dns_timeout 292.27 years
-dns_timeout 29.227 decades
-
-
+# check support for the maximum value (expressed in each supported time unit)
+# XXX: Use a 32-bit maximum to avoid breaking tests on 32-bit platforms.
+# TODO: Generate these test lines based on the target build platform.
+dns_timeout 2147483647854 milliseconds
+dns_timeout 2147483647 seconds # 2^31-1
```

Comment:
In my tests on a 32-bit platform, Squid parsers `dns_timeout 2147483649` without errors. The same Squid throws on `max_stale 2147483649`. Perhaps 32 bit platforms differ. @kinkie, does Squid fail on `dns_timeout 2147483649` in your tests?

The two directives have different underlying types (which probably explains why @eduard-bagdasaryan used two directive to test these maximums instead of one): for `dns_timeout` is `time_msec` while `max_stale` is `time_t`. The same "use maximum value" principle applies to both, but that maximum may differ for each type.